backend/globaleaks/handlers/rtip.py

```diff
@@ -399,7 +398,6 @@ def put(self, tip_id):
 
         # TODO A 202 is returned regardless of whether or not an update was performed.
         self.set_status(202)  # Updated
```

Comment:
It would be nice to see an example of the proper handling of failure codes in a handler like this one.